cfme/cloud/host_aggregates.py

```diff
@@ -1,37 +1,84 @@
+""" Page functions for Host Aggregates pages
+"""
 import attr
 from navmazing import NavigateToAttribute
-from widgetastic.widget import Text
-from widgetastic.widget import View
 from widgetastic_patternfly import BootstrapNav
+from widgetastic_patternfly import BreadCrumb
+from widgetastic_patternfly import Button
 from widgetastic_patternfly import Dropdown
+from widgetastic_patternfly import View
 
-from cfme.common import BaseLoggedInPage
+from cfme.base.ui import BaseLoggedInPage
+from cfme.common import Taggable
+from cfme.common import TaggableCollection
+from cfme.exceptions import ItemNotFound
 from cfme.modeling.base import BaseCollection
 from cfme.modeling.base import BaseEntity
 from cfme.utils.appliance.implementations.ui import CFMENavigateStep
+from cfme.utils.appliance.implementations.ui import navigate_to
 from cfme.utils.appliance.implementations.ui import navigator
+from cfme.utils.providers import get_crud_by_name
 from widgetastic_manageiq import Accordion
+from widgetastic_manageiq import BaseEntitiesView
 from widgetastic_manageiq import ItemsToolBarViewSelector
 from widgetastic_manageiq import ManageIQTree
+from widgetastic_manageiq import PaginationPane
 from widgetastic_manageiq import Search
+from widgetastic_manageiq import SummaryTable
+from widgetastic_manageiq import Table
+from widgetastic_manageiq import Text
 
 
-class HostAggregatesToolbar(View):
-    configuration = Dropdown('Configuration')
+class HostAggregatesView(BaseLoggedInPage):
+    @property
+    def in_host_aggregates(self):
+        return (
+            self.logged_in_as_current_user and
+            self.navigation.currently_selected == ['Compute', 'Clouds', 'Host Aggregates']
+        )
+
+
+class HostAggregatesToolBar(View):
     policy = Dropdown('Policy')
     download = Dropdown('Download')
+    configuration = Dropdown('Configuration')
     view_selector = View.nested(ItemsToolBarViewSelector)
 
 
-class HostAggregatesView(BaseLoggedInPage):
+class HostAggregatesEntities(BaseEntitiesView):
+    table = Table("//div[@id='gtl_div']//table")
+    # todo: remove table and use entities instead
+
```

Comment:
**Required:**
please refer this [doc](https://github.com/ManageIQ/integration_tests/blob/master/docs/guides/dev_guide.rst) and update ```todo``` as per instructions.

Reply:
nice catch, it was actually useless line of code

Reply:
And there could be more stuff that doesn't make sense in this code, which I don't see.